profiles/common/modules/custom/nexteuropa_token/modules/nexteuropa_token_ckeditor/nexteuropa_token_ckeditor.module

```diff
@@ -196,3 +196,15 @@ function nexteuropa_token_ckeditor_token_filter_filter_tokens($text, $filter, $f
 
   return _token_filter_filter_tokens($text, $filter, $format, $langcode, $cache, $cache_id);
 }
+
+/**
+ * Implements hook_custom_theme().
+ */
+function nexteuropa_token_ckeditor_custom_theme() {
+  if (strpos(current_path(), 'nexteuropa/token-ckeditor') === 0) {
+    $theme = variable_get('node_admin_theme', '0') ?
+      variable_get('admin_theme_default', 'seven') :
+      variable_get('theme_default', 'ec_resp');
```

Comment:
@gillesdeudon actually if i'm not wrong ec_resp is currently the base theme if you install a new NE instance, i see it here: https://github.com/ec-europa/platform-dev/blob/d7e625e773df942543a8d970c9151a27bbed584d/profiles/common/modules/features/multisite_settings/multisite_settings_core/multisite_settings_core.strongarm.inc#L140 and because multisite_settings_core is a dependency of multisite_drupal_standard profile (https://github.com/ec-europa/platform-dev/blob/d7e625e773df942543a8d970c9151a27bbed584d/profiles/multisite_drupal_standard/multisite_drupal_standard.info#L118) you can be sure this theme is currently the base theme on every subsites. The usage of variable_get is required like this because if something else is the theme_default, it has to be used, otherwise the 'common' one: ec_resp -- is it clear like this? :)